jmxtrans-core/src/main/java/com/googlecode/jmxtrans/jobs/ServerJob.java

```diff
@@ -41,8 +47,17 @@ public void execute(JobExecutionContext context) throws JobExecutionException {
 
 		JMXConnector conn = null;
 		JMXConnectionParams connectionParams = null;
+
 		try {
-			connectionParams = new JMXConnectionParams(server.getJmxServiceURL(), server.getEnvironment());
+			JMXServiceURL jmxUrl;
```

Comment:
It seems that we now have the concept of a `ServerJob` and a `LocalJob`. Could we reflect this in the class hierarchy instead of increasing the cyclomatic complexity of `ServerJob` ?